studio/src/hooks/use-window-size.ts

```diff
@@ -32,6 +32,7 @@ export default function useWindowSize() {
   return {
     windowSize,
     isMobile: typeof windowSize?.width === "number" && windowSize?.width < 768,
+    isTablet: typeof windowSize?.width === "number" && windowSize?.width < 1024,
```

Comment:
Is that aligned with tailwind?

Reply:
Yes, but we don't need it, cleaned it up